tests/ZiraLink.Api.Application.UnitTests/Services/CustomerServiceTest.cs

```diff
@@ -89,52 +91,310 @@ public async Task CreateLocallyAsync_WhenEverythingIsOk_ShouldBeSucceeded(string
             Assert.Equal(username, createdRow.Username);
             Assert.Equal(email, createdRow.Email);
             Assert.Equal(name, createdRow.Name);
-            Assert.Equal(family, createdRow.Family);  
+            Assert.Equal(family, createdRow.Family);
         }
 
         [Theory]
         [InlineData("10", "", "TestEmail@email.com", "Test", "Testi")]
-        public async Task CreateLocallyAsync_WhenUsernameIsEmpty_ShouldBeFailed(string externalId, string username, string email, string name, string family)
+        public async Task CreateCustomerLocally_WhenUsernameIsEmpty_ShouldBeFailed(string externalId, string username, string email, string name, string family)
         {
-            var mockConfiguration = new Mock<IConfiguration>();
-
-            mockConfiguration.Setup(m => m["ZIRALINK_URL_IDS"]).Returns("https://ids.ziralink.local:5001");
+            var mockHttpTools = new Mock<IHttpTools>();
 
-            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockConfiguration.Object);
+            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockHttpTools.Object);
 
             var exception = await Assert.ThrowsAsync<ArgumentNullException>(() => customerService.CreateLocallyAsync(externalId, username, email, name, family, CancellationToken.None));
             Assert.Equal("username", exception.ParamName);
         }
-        
+
         [Theory]
         [InlineData("10", "TestUser", "TestEmail@email.com", "", "Testi")]
-        public async Task CreateLocallyAsync_WhenNameIsEmpty_ShouldBeFailed(string externalId, string username, string email, string name, string family)
+        public async Task CreateCustomerLocally_WhenNameIsEmpty_ShouldBeFailed(string externalId, string username, string email, string name, string family)
```

Comment:
I would suggest keeping the first part `CreateLocallyAsync` as it is the method's name

Reply:
Done.